integration/k8s_fixture_test.go

```diff
@@ -181,26 +179,17 @@ func (f *k8sFixture) ClearNamespace() {
 }
 
 func (f *k8sFixture) setupNewKubeConfig() {
-	kubeConfigPath, exists := os.LookupEnv("KUBECONFIG")
-	if !exists {
-		usr, _ := user.Current()
-		dir := usr.HomeDir
-		kubeConfigPath = filepath.Join(dir, ".kube", "config")
-	}
-
-	current, err := ioutil.ReadFile(kubeConfigPath)
+	cmd := exec.CommandContext(f.ctx, "kubectl", "config", "view", "--minify")
+	current, err := cmd.Output()
 	if err != nil {
 		f.t.Fatalf("Error reading KUBECONFIG: %v", err)
 	}
 
-	f.tempDir.WriteFile("config", string(current))
-	f.fixture.tiltEnviron["KUBECONFIG"] = f.tempDir.JoinPath("config")
+	f.kubeconfigPath = f.tempDir.JoinPath("config")
+	f.tempDir.WriteFile(f.kubeconfigPath, string(current))
+	f.fixture.tiltEnviron["KUBECONFIG"] = f.kubeconfigPath
 	f.fixture.tiltEnviron["TILT_K8S_EVENTS"] = "true"
-	f.usingOverriddenConfig = true
-}
-
-func (f *k8sFixture) createUser() {
-	f.runCommandSilently("kubectl", "apply", "-f", "access.yaml")
+	log.Printf("New kubeconfig: %s", f.kubeconfigPath)
```

Comment:
👀

Reply:
this was deliberate!

Reply:
![https://media1.tenor.com/images/a4b32543e32fcbb8cf42fbebd146e38e/tenor.gif?itemid=5502727](https://media1.tenor.com/images/a4b32543e32fcbb8cf42fbebd146e38e/tenor.gif?itemid=5502727)